iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/SpecificVersionTagCheck.java

```diff
@@ -48,6 +48,11 @@ private static void checkBody(CheckContext ctx, Body body) {
   }
 
   private static void checkFromInstruction(CheckContext ctx, FromInstruction fromInstruction, Set<String> encounteredAlias) {
+    var alias = fromInstruction.alias();
+    if (alias != null) {
+      encounteredAlias.add(alias.alias().value());
+    }
```

Comment:
Please double-check if the implementation is correct because the ticket description says: `But, the addition of the alias should not be done at the top of this method, as we don’t want to miss code like: `.
Please also add more tests cases to be on the safe side.